src/olympia/shelves/models.py

```diff
@@ -2,8 +2,8 @@
 
 from olympia.amo.models import ModelBase
 
-SHELF_TYPES = (
-    'category', 'collection', 'extension', 'recommended', 'search', 'theme')
+SHELF_TYPES = ('categories', 'collections', 'extension',
+               'recommendations', 'search', 'theme')
```

Comment:
I wanted to revise the types to match the name of the API being used to avoid any confusion when creating shelves. I would've left it alone if it was already in use, but since it's not, I figured now's the time to make the update.